### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by bbakiko** [AIMLab-UBC/MIDL2020#16]
Hi, thank you for your great work. I'd like to know if the dataset is now available. really want to give it a try :)

**Comment by kpup1710** [comment]
Is the dataset available yet?

**Comment by Giovana-Rodrigues** [comment]
Hello! Is the dataset available yet? I would like to use it in my final project

**Comment by AIMLab-UBC** [comment]
Dataset is available through the OCEAN Challenge on Kaggle.

**Comment by Giovana-Rodrigues** [comment]
Yes, I saw that there are around 1k images. However, in your article you mention that you used around 2k images. Would it be possible for me to have the entire dataset?

**Comment by AIMLab-UBC** [comment]
I am not sure which article you are referring to. The OCEAN dataset is the largest public dataset on ovarian carcinomas. We may release an even larger set after publishing the OCEAN challenge results.

**Comment by Giovana-Rodrigues** [comment]
Got it! Could I ask if you have an estimated release date for this larger dataset? I would really like to use it in my university final project. We're implementing the models in the cloud to evaluate performance, but we wanted to test it with a larger number of images. The idea is to create an endpoint where the doctor can upload patient images, and after the model processes them, it would return the associated cancer subtype along with the accuracy rate for that subtype.

**Comment by Giovana-Rodrigues** [comment]
Sorry for the delay—I was double-checking the information regarding the article.

Basically, within Kaggle, after scrolling down a bit in the "Overview" section, there's a part called "How to Cite This Challenge in Publications." It contains two citations pointing to different articles. The first article discusses the competition and its objectives, as well as topics like the dataset.

This article has a section in the "Results" called "Dataset." In this paragraph, it mentions that the competition contains "2,438 images distributed across three distinct sets: training, public test, and private test, sourced from 24 centers that were mainly part of the ovarian tumor tissue analysis (OTTA) consortium [...]"

However, in the competition data, I only have access to around 1k images. Maybe I misunderstood something? I would be very grateful if you could explain.
